### DISCUSSION TITLE
Standard Charge amounts - Can we use a range for our DRGs

### DISCUSSION THREAD

**Discussion by rlarsonat** [CMSgov/hospital-price-transparency#41] [Q&A] [answered]
I know that in previous versions, it was stated that the standard charge could be listed as a range which would then translate in the contracted rates potentially being a range.  This would more likely be for those items billed using the MS-DRG where the charges could vary from person to person but the contracted rate may not.  Is this still allowed?

**Comment by RidenourK** [comment, marked as answer]
@rlarsonat. Thank you for your question. 

Please refer to the CSV and JSON documentation for information about how to encode data for standard charges and coding data elements and their valid values. Ranges are not specified in the CSV and JSON documentation and values encoded incorrectly will generate a deficiency. 

Please see the CSV and JSON examples for examples of how to encode data in this scenario.
